ompi/mca/osc/ucx/osc_ucx_comm.c

```diff
@@ -380,6 +380,13 @@ int ompi_osc_ucx_put(const void *origin_addr, int origin_count, struct ompi_data
         }
     }
 
+    if (!((module->win_info_array[target]).rkey_init) && (target_count > 0)) {
+        OSC_UCX_VERBOSE(1, "target window has no rkey or has zero size");
```

Comment:
"window with non-zero length does not have an rkey"

Reply:
fixed